### PR TITLE
templates: minor wording fixes

### PR DESCRIPTION
**Pull request by chriz-uniba**

### CODE REVIEW

**Review by tmorrell — APPROVED** [review]
Thanks!

**Comment by chokribr** [comment]
@kpsherva need one more aproval for this doc minor change

File: docs/customize/look-and-feel/templates.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ THEME_FRONTPAGE_TEMPLATE = 'invenio_app_rdm/frontpage.html'
 
 There are 2 options to change this template:
 
-1. Change the configuration value setting the new template path. In this way, you can organize your template's files as you see fit.
-2. Create a file with same path relative to `templates` as the original. In this other way, you can keep the template's files as structured and organized in InvenioRDM.
+1. Change the configuration value setting to the new template path. In this way, you can organize your template's files as you see fit.
```
Comment (fenekku) [inline docs/customize/look-and-feel/templates.md:18]:
```suggestion
1. Change the configuration value setting to the new template path. This way, you can place and name your template file as you see fit.
```

File: docs/customize/look-and-feel/templates.md [hunk 2]
```diff
@@ -15,8 +15,8 @@ THEME_FRONTPAGE_TEMPLATE = 'invenio_app_rdm/frontpage.html'
...
+2. Create a file with the same path relative to `templates` as the original. In this way, you can keep the template's files as structured and organized in InvenioRDM.
```
Comment (fenekku) [inline docs/customize/look-and-feel/templates.md:19]:
```suggestion
2. Create a file with the same path relative to `templates` as the original. This way, you can keep the same structure and organization as InvenioRDM for your template file.
```

**Review by fenekku — APPROVED** [review]
Thanks for this. Can be merged after minor comments are addressed.

**Comment by chriz-uniba** [comment]
Comments are addressed now, thanks.

**Pull request merged**